src/Nancy/ErrorHandling/DefaultStatusCodeHandlerResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Nancy.ErrorHandling
+{
+    /// <summary>
+    /// Default error handler result
+    /// </summary>
+    public class DefaultStatusCodeHandlerResult

```

Comment:
This is just an implementation detail of the `DefaultStatusCodeHandler`, right? Maybe this should just be a private class?